contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryLogicC2_3.sol

```diff
@@ -78,7 +78,10 @@ contract AutomationRegistryLogicC2_3 is AutomationRegistryBase2_3 {
     uint96 balance = _updateTransmitterBalanceFromPool(from, s_hotVars.totalPremium, uint96(s_transmittersList.length));
     s_transmitters[from].balance = 0;
     s_reserveAmounts[IERC20(address(i_link))] = s_reserveAmounts[IERC20(address(i_link))] - balance;
-    i_link.transfer(to, balance);
+    bool transferStatus = i_link.transfer(to, balance);
```

Comment:
why it won't fail in CI bc of you did not generate wrappers?

Reply:
hmm, can you elaborate on this?

Reply:
bc the contract is changed, don't you need to regenerate Go wrappers?

Reply:
wow this is _super_ interesting. My guess is that because the link token always returns `true`, the optimizer considers this a "dead path" and essentially removes it, making the current bytecode match the old bytecode.

Reply:
I ran wrappers generation locally, and this change doesnt change any wrappers. Correct me if I need other steps..

Reply:
If LINk never returns false, do we still need to check the status? our automation codebase has the tradition of checking the status :)

Reply:
@RyanRHall @FelixFan1992 If no objection, I will abandon this PR since checking the bool return is not needed for LINK.